process/peer/process.go

```diff
@@ -827,7 +874,10 @@ func (vs *validatorStatistics) updateValidatorInfo(validatorList []sharding.Vali
 			newRating = vs.rater.ComputeIncreaseValidator(shardId, peerAcc.GetTempRating())
 		}
 
-		peerAcc.SetTempRating(newRating)
+		if epoch >= vs.ratingEnableEpoch {
```

Comment:
So, the other stuff above will be affected. Only the rating will be kept unchanged? peerAcc.IncreaseLeaderSuccessRate(1), peerAcc.SetConsecutiveProposerMisses(0), peerAcc.AddToAccumulatedFees(leaderAccumulatedFees), peerAcc.IncreaseValidatorSuccessRate(1), peerAcc.DecreaseValidatorSuccessRate(1)

Reply:
yes. those informations are needed to distribute rewards and all others.

Reply:
will change to do only the positive changes - no negative. It is better for the well behaving validators.